doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -14,7 +14,7 @@ syntax follows the schema:
 
 .. cmd:: Scheme @ident := Induction for @ident Sort sort {* with @ident := Induction for @ident Sort sort}
 
-where each `ident'ᵢ` is a different inductive type identifier 
+where each `ident'ᵢ` is a different inductive type identifier
```

Comment:
Indentation seems wrong here (it was aready before this PR, but while we are at it...)